Traceback (most recent call last):
  File "/home/doctorlab/Documents/python/rpicamlab/main.py", line 1, in <module>
    from picamera2 import Picamera2
  File "/usr/lib/python3/dist-packages/picamera2/__init__.py", line 11, in <module>
    from .picamera2 import Picamera2, Preview
  File "/usr/lib/python3/dist-packages/picamera2/picamera2.py", line 30, in <module>
    from picamera2.encoders import Encoder, H264Encoder, MJPEGEncoder, Quality
  File "/usr/lib/python3/dist-packages/picamera2/encoders/__init__.py", line 5, in <module>
    from .encoder import Encoder, Quality
  File "/usr/lib/python3/dist-packages/picamera2/encoders/encoder.py", line 12, in <module>
    from ..request import _MappedBuffer
  File "/usr/lib/python3/dist-packages/picamera2/request.py", line 13, in <module>
    import simplejpeg
  File "/usr/lib/python3/dist-packages/simplejpeg/__init__.py", line 1, in <module>
    from ._jpeg import encode_jpeg, encode_jpeg_yuv_planes
  File "simplejpeg/_jpeg.pyx", line 1, in init simplejpeg._jpeg
ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
